kf_lib_data_ingest/common/misc.py

```diff
@@ -87,7 +88,7 @@ def iterate_pairwise(iterable):
     return zip(a, b)
 
 
-def numeric_to_str(val, replace_na=False, na=None):
+def convert_to_str(val, replace_na=False, na=None):
```

Comment:
Maybe this should be called `convert_to_downcasted_str` so that someone is aware that there is something more than just conversion to strs going on.

Reply:
√